server/log_generator.py

```
@@ -54,6 +58,31 @@
+    global _server_start_time
+    print("[LOG GENERATOR] Waiting for server to be ready...")
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            response = requests.get(url, timeout=2)
+            if response.status_code == 200:
+                _server_start_time = time.time()
+                print("[LOG GENERATOR] Server is ready. Starting.")
+                return True
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)